packages/axios-extension/src/abap/types/adt-types.ts

```diff
@@ -95,3 +95,9 @@ export type AdtTransportStatus = 'S' | 'E';
 // the <DLVUNIT/> element contain text that indicate it is local package.
 // No transport number required for deploying to local pacakge.
 export const LocalPackageText = ['LOCAL_PACKAGE', 'LOCAL'];
+
+export interface PackageInfo {
+    uri: string;
+    type: 'DEVC/K';
```

Comment:
Why a hardcode value for `type`? Maybe a constant would do? What's `DEVC/K`?

Reply:
Used string type instead.